### PR TITLE
[zh-cn]: update the translation of URL `href` and `search` property

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/web/api/url/href/index.md [hunk 1]
```diff
@@ -1,30 +1,25 @@
...
 
 ## 示例
 
 ```js
-var url = new URL("https://developer.mozilla.org/zh-CN/docs/Web/API/URL/href");
-var result = url.href; // Returns: 'https://developer.mozilla.org/zh-CN/docs/Web/API/URL/href'
+const url = new URL(
+  "https://developer.mozilla.org/zh-CN/docs/Web/API/URL/href",
+);
+console.log(url.href); // 输出：'https://developer.mozilla.org/en-US/docs/Web/API/URL/href'
```
Comment (T34-active) [inline files/zh-cn/web/api/url/href/index.md:22]:
```suggestion
console.log(url.href); // 输出：'https://developer.mozilla.org/zh-CN/docs/Web/API/URL/href'
```

**Review by T34-active — COMMENTED** [review]
fix output link

File: files/zh-cn/web/api/url/search/index.md [hunk 2]
```diff
@@ -1,34 +1,27 @@
...
-一个 {{domxref("USVString")}}.
-
-## 例子
+## 示例
 
 ```js
-var url = new URL(
-  "https://developer.mozilla.org/zh-CN/docs/Web/API/URL/search?q=123",
+const url = new URL(
+  "https://developer.mozilla.org/en-US/docs/Web/API/URL/search?q=123",
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/url/search/index.md:22]:
```suggestion
  "https://developer.mozilla.org/zh-CN/docs/Web/API/URL/search?q=123",
```

File: files/zh-cn/web/api/url/href/index.md [hunk 3]
```diff
@@ -1,30 +1,25 @@
...
 slug: Web/API/URL/href
+l10n:
+  sourceCommit: 4de6f76bbfd76229db78ffb7d52cf6b4cb9f31f8
 ---
 
-{{ApiRef("URL API")}}
+{{ApiRef("URL API")}} {{AvailableInWorkers}}
 
-{{domxref("URL")}} 接口的 **`href`** 属性是一个包含完整 URL 的 {{domxref("USVString")}} 值。
+{{domxref("URL")}} 接口的 **`href`** 属性是包含整个 URL 的完整信息字符串。
```
Comment (yin1999) [inline files/zh-cn/web/api/url/href/index.md:10]:
```suggestion
{{domxref("URL")}} 接口的 **`href`** 属性是包含整个 URL 的字符串。
```

File: files/zh-cn/web/api/url/search/index.md [hunk 4]
```diff
@@ -1,34 +1,27 @@
...
 slug: Web/API/URL/search
+l10n:
+  sourceCommit: 4de6f76bbfd76229db78ffb7d52cf6b4cb9f31f8
 ---
 
-{{ApiRef("URL API")}}
+{{ApiRef("URL API")}} {{AvailableInWorkers}}
 
-{{domxref("URL")}} 接口的 **`search`** 属性是一个搜索字符串，也称为查询字符串，这是一个字符串包含一个 `'?'` 后面跟着 URL 的参数。
+{{domxref("URL")}} 接口的 **`search`** 属性表示包含一个 `'?'` 后跟随 URL 参数的搜索字符串（也被称为*查询字符串*）。
```
Comment (yin1999) [inline files/zh-cn/web/api/url/search/index.md:10]: 原翻译的行文格式更易于理解

File: files/zh-cn/web/api/url/search/index.md [hunk 5]
```diff
@@ -1,34 +1,27 @@
...
-## 例子
+## 示例
 
 ```js
-var url = new URL(
+const url = new URL(
   "https://developer.mozilla.org/zh-CN/docs/Web/API/URL/search?q=123",
 );
-var queryString = url.search; // Returns:"?q=123"
+console.log(url.search); // 输出 "?q=123"
```
Comment (yin1999) [inline files/zh-cn/web/api/url/search/index.md:24]:
```suggestion
console.log(url.search); // 输出“?q=123”
```

File: files/zh-cn/web/api/url/search/index.md (hunk 4 above)
Comment (T34-active) [inline files/zh-cn/web/api/url/search/index.md:10]:
```suggestion
{{domxref("URL")}} 接口的 **`search`** 属性是一个搜索字符串，也称为查询字符串，这是一个字符串包含一个 `'?'` 后面跟着 URL 的参数。
```

File: files/zh-cn/web/api/url/search/index.md [hunk 6]
```diff
@@ -1,34 +1,27 @@
...
+title: URL：search 属性
 slug: Web/API/URL/search
+l10n:
+  sourceCommit: 4de6f76bbfd76229db78ffb7d52cf6b4cb9f31f8
 ---
 
-{{ApiRef("URL API")}}
+{{ApiRef("URL API")}} {{AvailableInWorkers}}
 
 {{domxref("URL")}} 接口的 **`search`** 属性是一个搜索字符串，也称为查询字符串，这是一个字符串包含一个 `'?'` 后面跟着 URL 的参数。
```
Comment (yin1999) [inline files/zh-cn/web/api/url/search/index.md:10]:
```suggestion
{{domxref("URL")}} 接口的 **`search`** 属性是一个搜索字符串，也称为查询字符串，这是一个包含 `'?'` 且其后跟着 URL 的参数的字符串。
```

**Review by yin1999 — APPROVED** [review]
:+1:

**Pull request merged**